last_login added to UserDefinitions.php

introspectionism opened the pull request:
Added the last_login value to the user endpoint.

This fixes #135.

luke- reviewed (changes requested):
@introspectionism Thanks for the contribution.

Can you please add this also to the documentation https://github.com/humhub/rest/blob/master/docs/swagger/user.yaml and the Changelog?

introspectionism commented:
@luke-

I am a complete noob when it comes to doing PRs. Actually, this is the first I've ever done. So, I am not sure how to do this. I will try a few things.

Here I updated Swagger (but, not sure how to push this to the correct branch):
https://github.com/introspectionism/rest/commit/d67f110db709e360a67c3383fa9ae16c32423367

And here I updated the changelog:
https://github.com/introspectionism/rest/commit/d869fec91031232a26c84900e80c78b5300e7a5e

I might be messing up things here. Sorry about that.

introspectionism commented:
I have created a new branch, with all the requested changes:
https://github.com/introspectionism/rest/tree/fix/135-last_login/
